docs/en/defined-terms.rst

```diff
@@ -5,12 +5,41 @@
 defined-terms.rst
 +++++++++++++++++++++++++++
 
-[What is it]
-    
-[What it is usefull for]
-    
-[Example]
-    
+Below are the description of acronyms and definitions which are useful for further insights into topics that complement the it-wallet and the interacting components.
+
+.. list-table:: 
+  :widths: 20 80
+  :header-rows: 1
+
+  * - **Acronym**
+    - **Description**
+  * - **OID4VP**
+    - OpenID for Verifiable Presentation 
+  * - **PID** 
+    - Person Identification Data
+  * - **VC**
+    - Verifiable Credential
+  * - **VP**
+    - Verifiable Presentation
+  * - **API**
+    - Application Programming Interface
+
+
+Acronyms
+--------
+
+.. list-table:: 
+  :widths: 20 80
+  :header-rows: 1
+
+  * - **Definition**
+    - **Description**
+  * - **WALLET**
+    - Mobile app that manages, stores, and protects the Verifiable Credentials of a holder and allows the presentation to a Relying Party
+  * - **RELYING PARTY**
+    - Entity that receives one or more VC/VP from an instance of a Wallet and processes them
```

Comment:
VC/VP -> VP

instance of a Wallet -> Wallet Instance

Reply:
Done